src/makeCalculator.js

```diff
@@ -37,7 +37,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  function add(num) {
+    result += num;
+  }
+
+  function subtract(num) {
+    result -= num;
+  }
+
+  function multiply(num) {
+    result *= num;
+  }
+
+  function divide(num) {
+    result /= num;
+  }
+
+  function reset() {
+    result = 0;
+
+    return calculator;
+  }
+
+  function operate(callback, num) {
+    callback(num);
+
+    return calculator;
+  }
```

Comment:
let's put this function to object? no need to declare them above and then paste them below